Topics/Development_Process/Serverless_AWS_Lambda.md

```diff
@@ -0,0 +1,69 @@
+2. **Scalability**: Because it is not up to the developer to allocate and maintain servers, developers do not have to worry about scaling servers up or down, as it is up to the cloud provider to do so.
+3. **Easier maintenance**: The developer/user does not need to manage their backend services, as it is up to the cloud provider to maintain the servers. With containers or other deployment options, developers are responsible for updating and maintaining them.
+4. **Deployment time**: Serverless functions finish deploying as soon as their code is updated, making them generally faster than containers, or virtual machines.
+
+## Disadvantages of Serverless Computing
+
+There are some situations where you might not want to use serverless computing.
+
+1. **Boot-up time**: Serverless services are not constantly running, so when you make a request to code that hasn't been used in a while, you could face performance impacts.
+2. **Long running processes**: Since you pay for what you use, it is usually not cost-effective to use serverless for longer running processes. In these cases, you might want to consider other options to run your backend services on.
+
+## AWS Lambda
+
+Lambda is a serverless compute service offered by Amazon Web Services (AWS). It allows you to run backend code triggered by events, such as other AWS services or your own services, without needing to provision or manage servers. And, you only pay for each request you make. In the free tier for AWS Lambda, you have access to 1 million free requests per month.
+
+### Creating Lambda Functions in the AWS Console
+
+To use Lambda, you must have an AWS account with an administrative user. Follow the instructions [here](https://portal.aws.amazon.com/billing/signup) on how to sign up.
+
+To create a simple 'Hello World' function using Node.js or Python, see the tutorial, see the tutorial, [Create a Lambda Function with the console](https://docs.aws.amazon.com/lambda/latest/dg/getting-started.html#getting-started-create-function). Interpreted languages are easier to start off with since you can edit the code directly in the AWS console's code editor.
+
+### Invoking Lambda Functions from the AWS Console
+
+Once you have created your function in the console, there are many ways to invoke it. To invoke your function through the AWS console, see the tutorial, [Invoking the Lambda Function using the console](https://docs.aws.amazon.com/lambda/latest/dg/getting-started.html#get-started-invoke-manually). You can also invoke it through the AWS CLI, HTTPs endpoints, AWS SDK, other AWS services, and more. See [Invoking Lambda Functions](https://docs.aws.amazon.com/lambda/latest/dg/lambda-invocation.html).
+
+Overall, Lambda functions are very versatile and can be a great alternative for running backend code.
```

Comment:
Great tutorial! I like that it's not super in depth, but it still gives a great high-level intuition and overview. Also big fan of the external resources linked in the actual article instead of just listing them at the end :)

Reply:
Thanks! Do you think I should add more specific resources in the article? Such as a lower level explanation of something, etc.

Reply:
That makes sense, and I agree. Thanks for the input!